FirebaseSessions/Sources/NetworkInfo.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+protocol NetworkInfoProtocol {
+  var mobileCountryCode: String? { get }
+
+  var mobileNetworkCode: String? { get }
+}
+
+class NetworkInfo: NetworkInfoProtocol {
```

Comment:
Should the network info extraction be moved to this class?

Reply:
I didn't do that so that we can share that with Sessions and FirePerf

Reply:
Sounds good.